storage/innobase/os/os0proc.cc

```diff
@@ -186,4 +176,11 @@ os_mem_free_large(
 		os_total_large_mem_allocated -= size;
 	}
 #endif
+#if defined HAVE_LINUX_LARGE_PAGES && defined UNIV_LINUX
+	if (my_use_large_pages) {
+		/* note accounting will be off if we fell though to
+		conventional memory on allocation */
+		os_total_large_mem_allocated -= size;
+	}
+#endif /* HAVE_LINUX_LARGE_PAGES && UNIV_LINUX */
```

Comment:
Yes, you just moved some code around, but while doing it you might clean it up.
Doesn’t `HAVE_LINUX_LARGE_PAGES` imply `UNIV_LINUX`?